sections/merge-accounts/burn/BurnActionBox.tsx

```diff
@@ -0,0 +1,240 @@
+import { useMemo, useEffect, FC, useCallback } from 'react';
+import styled from 'styled-components';
+import { Trans, useTranslation } from 'react-i18next';
+import { Svg } from 'react-optimized-image';
+import { useRouter } from 'next/router';
+import Wei, { wei } from '@synthetixio/wei';
+
+import Connector from 'containers/Connector';
+import StructuredTab from 'components/StructuredTab';
+import Etherscan from 'containers/BlockExplorer';
+import NavigationBack from 'assets/svg/app/navigation-back.svg';
+import GasSelector from 'components/GasSelector';
+import { BurnActionType } from 'store/staking';
+import {
+	ModalContent as TxModalContent,
+	ModalItemTitle as TxModalItemTitle,
+	ModalItemText as TxModalItemText,
+	ModalItemSeperator as TxModalItemSeperator,
+	NoTextTransform,
+	GlowingCircle,
+	IconButton,
+} from 'styles/common';
+import { StyledCTA } from 'sections/staking/components/common';
+import {
+	FormContainer,
+	InputsContainer,
+	SettingsContainer,
+	SettingContainer,
+	TxModalItem,
+	FormHeader,
+} from 'sections/merge-accounts/common';
+import TxConfirmationModal from 'sections/shared/modals/TxConfirmationModal';
+import { formatCryptoCurrency, formatNumber } from 'utils/formatters/number';
+import { DEFAULT_FIAT_DECIMALS } from 'constants/defaults';
+import ROUTES from 'constants/routes';
+import Currency from 'components/Currency';
+import { CryptoCurrency, Synths } from 'constants/currency';
+import { parseSafeWei } from 'utils/parse';
+import { getStakingAmount } from 'sections/staking/components/helper';
+import useBurnTx from 'sections/staking/hooks/useBurnTx';
+
+import { TxWaiting, TxSuccess } from './Tx';
+
+const BurnTab: FC = () => {
+	const { t } = useTranslation();
+
+	const tabData = useMemo(
+		() => [
+			{
+				title: t('merge-accounts.burn.title'),
+				tabChildren: <BurnTabInner />,
+				key: 'main',
+				blue: true,
+			},
+		],
+		[t]
+	);
+
+	return (
+		<StructuredTab singleTab={true} boxPadding={20} tabData={tabData} setPanelType={() => null} />
+	);
+};
+
+const BurnTabInner: FC = () => {
+	const { t } = useTranslation();
+	const { connectWallet } = Connector.useContainer();
+
+	const router = useRouter();
+	const { blockExplorerInstance } = Etherscan.useContainer();
+
+	const {
+		debtBalance,
+		sUSDBalance,
+		issuableSynths,
+		txn,
+		onBurnChange,
+		onBurnTypeChange,
+		error,
+		txModalOpen,
+		setTxModalOpen,
+		setGasPrice,
+		isWalletConnected,
+		targetCRatio,
+		SNXRate,
+		currentCRatio,
+	} = useBurnTx();
+
+	const stakeInfo = useCallback(
+		(burnAmount: Wei): Wei => wei(getStakingAmount(targetCRatio, burnAmount, SNXRate)),
+		[SNXRate, targetCRatio]
+	);
+
+	const burnAmount = debtBalance.gt(sUSDBalance) ? wei(sUSDBalance) : debtBalance;
```

Comment:
@vbstreetz we want to burn max? So let's take the sUSD balance as the burn amount and allow the burn when balance >= debt. Because the way you do it could lead to issue if the debt shifts while the transaction is being mined.

Reply:
@clementbalestrat does this need to be updated as well? https://github.com/Synthetixio/staking/blob/dev/sections/staking/components/BurnTab/BurnTab.tsx#L50-L51